arches_querysets/datatypes/file.py

```diff
@@ -0,0 +1,70 @@
+import copy
+
+from django.utils.translation import get_language
+
+from arches.app.datatypes import datatypes
+from arches.app.utils.i18n import rank_label
+
+
+class FileListDataType(datatypes.FileListDataType):
+    def transform_value_for_tile(self, value, *, languages, **kwargs):
+        if not value:
+            return value
+
+        stringified_list = ",".join([file_info.get("name") for file_info in value])
+        final_value = self.transform_value_for_tile(
+            stringified_list, languages=languages, **kwargs
+        )
+
+        for file_info in final_value:
+            for key, val in file_info.items():
+                if key not in {"altText", "attribution", "description", "title"}:
```

Comment:
is there a way to derive these programatically?

Reply:
ugh yeah this is probably a bigger conversation but IMO those fields should exist in a defaultconfig somewhere so that we can leverage `i18n_properties`. But that's probably out-of-scope for this